forms/tests/test_utils.py

```diff
@@ -40,8 +40,10 @@ def test_form_cloning(basic_template_form):
     new_fieldtype_count = FieldType.objects.all().count()
 
     assert new_form.id != basic_template_form
-    assert new_section_count == section_count * 2
-    assert new_field_count == field_count * 2
+    # Cloning basic template form should add 7 sections.
+    assert new_section_count == section_count + 7
```

Comment:
It might be beneficial to add these comments in the assert as the second argument like `assert new section_count == section count + 7, "Cloning basic template form should add 7 sections"`
https://docs.pytest.org/en/stable/how-to/assert.html

Reply:
Looks like handy feature, wasn't aware of that. Definitely going to add something to this case :)